### PR TITLE
[PTQ][TORCH] Fix NNCFNetwork creation

### PR DESCRIPTION
**Pull request by daniil-lyakhov**
### Changes

NNCFNetwok creation is fixed for Torch PTQ

### Reason for changes

To enable Torch PTQ validation

### Related tickets

109662

### Tests

`tests/torch/ptq/test_helpers.py`

### CODE REVIEW

**Comment by daniil-lyakhov** [comment]
@vshampor, please do not merge without @alexsu52 approve

File: tests/torch/ptq/test_helpers.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 Intel Corporation
```
Comment (AlexanderDokuchaev) [inline tests/torch/ptq/test_helpers.py:1]: Looks like duplicate of `test_quantize_model_helpers.py`

File: nncf/experimental/torch/quantization/quantize_model.py [hunk 2]
```diff
@@ -54,14 +59,16 @@ def create_dummy_forward_fn(dataset, device):
...
 
                 def send_to_device(tensor):
                     return tensor.to(device)
 
                 args = objwalk(args, is_tensor, send_to_device)
+                kwargs = objwalk(kwargs, is_tensor, send_to_device)
 
-            args = wrap_inputs(args)
-            retval = model(*args)
+            args, kwargs = wrap_inputs(args, kwargs)
```
[the remark this answers is not in the document]
Comment (alexsu52) [reply nncf/experimental/torch/quantization/quantize_model.py:68]: @daniil-lyakhov Thanks for the fix!

**Review by alexsu52 — APPROVED** [review]
LGTM

File: tests/torch/ptq/test_helpers.py (hunk 1 above)
Comment (daniil-lyakhov) [reply tests/torch/ptq/test_helpers.py]: You are right, thanks!

**Comment by daniil-lyakhov** [comment]
@vshampor please merge it if you don't have comments

**Pull request merged**